src/include/libpmemobj++/README.md

```diff
@@ -56,6 +56,24 @@ nvml::obj::transaction::automatic class requires C++17, so
 you need a more recent version for this to be available(GCC 6.1/Clang 3.7).
 It is recommended to use these or newer versions of GCC or Clang.
 
+### Standard notice ###
+Please note that the C++11 standard, section 6.8, states that a valid non-POD
```

Comment:
6.8? In my draft it's 3.8.

One other thing, in the standard, they don't mention POD per-se, just that objects need to be trivially default constructible? Or something along the lines at the same time mentioning object members... PODs definitely fall into that category, I'm just thinking out loud if we should reword it or not.

Reply:
I changed it, I think it's equally understandable.